GNUmakefile

```diff
@@ -32,10 +32,9 @@ smoke-tests: all
 .PHONY: update-graph
 update-graph: all
 	$Q set -xe; tmp=`mktemp -d`; \
-	trap "killall grout; wait; rm -rf $$tmp" EXIT; \
-	export GROUT_SOCK_PATH="$$tmp/sock"; \
+	trap "killall grout; wait;" EXIT; \
 	$(BUILDDIR)/grout -t & \
-	socat FILE:/dev/null UNIX-CONNECT:$$GROUT_SOCK_PATH,retry=10 && \
+	socat FILE:/dev/null ABSTRACT-CONNECT:grout.sock,retry=10 && \
```

Comment:
Could you preserve the ability to force a different name for the abstract socket. Possibly via a `GROUT_SOCK_NAME` variable.